otwcode/otwarchive
# AO3-2675 Prevent adding parent skins with blank titles

without errors.clear an additional error pops up that I can't trace. This should remove blank parents as acceptable parents

# Pull Request Checklist

  as the *first* thing in your pull request title (e.g. `AO3-1234 Fix thing`)
  until they are reviewed and merged before creating new pull requests.

## Issue

https://otwarchive.atlassian.net/browse/AO3-2675

## Purpose

What does this PR do?

This PR should fix an ongoing issue where it is possible to set a blank parent, which would result in the user experiencing CSS changes.

## Testing Instructions

How can the Archive's QA team verify that this is working as you intended?

Follow the original Steps to Reproduce in the ticket:

1. Log in
2. Hi, username! > My Dashboard > Skins > Create Site Skin
3. Fill in a title and some CSS, e.g. #header ul.primary { background: none blue; }
4. Select "Show" option next to "Advanced"
5. Select "Add parent skin" but do not fill out the field that appears
6. Select "Submit"

This will now result in an error with "You can't have an empty parent skin." popping up as the reason.

![AO3-2675_intended_error](https://user-images.githubusercontent.com/103412983/162695560-f027bf85-cff4-47cb-ae49-1def37d622a0.PNG)

## References

Are there other relevant issues/pull requests/mailing list discussions?

No.

## Credit

What name and pronouns should we use to credit you in the [Archive of Our Own's Release Notes](https://archiveofourown.org/admin_posts?tag=1)?

speckled_dragon or speck (she/her) is fine with me :)
